app/components/utils/user-table/template.hbs

```diff
@@ -13,7 +13,7 @@
     onScrolledToBottom=(action "onScrolledToBottom") as |body|
   }}
     {{#body.expanded-row as |row|}}
-      {{yield row}}
+      {{yield row }}
     {{/body.expanded-row}}
```

Comment:
I've also found a way to disable the cursor pointer on the entire row only in user management. You can add a class to the entire table here and scope the CSS accordingly. You will have to put this as a custom override that is specific to this part of the app.